static/app/views/settings/project/projectOwnership/addCodeOwnerModal.tsx

```diff
@@ -292,7 +292,6 @@ function ErrorMessage({
   errorJSON: {raw?: string} | null;
 }) {
   const codeMapping = codeMappings.find(mapping => mapping.id === codeMappingId);
-  const {integrationId, provider} = codeMapping as RepositoryProjectPathConfig;
```

Comment:
no need to force `RepositoryProjectPathConfig` type here, we have the `codeMapping &&` guard, and typescript can understand the refinement if we type `codeMapping.integrationId` instead of destructuring.